bench/workgen/workgen.cxx

```diff
@@ -949,6 +949,7 @@ ThreadRunner::op_run(Operation *op)
     Track *track;
     tint_t tint = op->_table._internal->_tint;
     WT_CURSOR *cursor;
+    WT_ITEM item;
```

Comment:
I think it's good practice to clear the memory `WT_CLEAR(item)` before using.  Otherwise, you might get some weird results using the item.